addons/web/static/tests/views/search_view_tests.js

```diff
@@ -343,6 +343,39 @@ QUnit.module('Search View', {
         testUtils.dom.click($('.o_facet_remove:first'));
         assert.strictEqual($('.o_searchview .o_searchview_facet .o_facet_values span').length, 0,
             'there should be no facet');
+
+        actionManager.destroy();
+    });
+
+    QUnit.test('remove a facet with backspace', function (assert) {
```

Comment:
can't you write this test with a view directly? we do not need to involve the action manager anymore, right?